test/Benchmarks/MapReduce/MapReduceBenchmarkConfig.cs

```diff
@@ -11,7 +11,7 @@ public MapReduceBenchmarkConfig()
             {
                 Run = {
                     LaunchCount = 1,
-                    TargetCount = 2,
+                    InvocationCount = 2,
```

Comment:
This is unrelatred, right?

Reply:
No, TargetCount disappeared and I believed this is the replacement. I was wrong - it should be `IterationCount`.

Reply:
Fixed in latest